src/classes/GE_FormRendererService.cls

```diff
@@ -218,12 +218,26 @@ public with sharing class GE_FormRendererService {
         return null;
     }
 
+    @AuraEnabled(Cacheable=true)
```

Comment:
Small nitpick, lets lower case the `cacheable`

Reply:
fixed